### ISSUE TITLE
Default file expiry in addition to max

### ISSUE THREAD

**Issue by wxcafe** [andreimarcu/linx-server#235]
Hey, thanks for this really cool piece of software

It would be great to be able to set a default file expiry in addition to the max one, i.e. I want to be able to upload files that stay up forever but most of the time I want a one hour or one day expiry and forget to change it before uploading, so I end up cancelling the upload and starting again.

**Comment by craSH** [comment]
Just came here to request this as well, +1 from me! Thanks for the great piece of software.

**Comment by andreimarcu** [comment]
@wxcafe @craSH 

Do you mean specifically on the upload page or even when you use something like `curl -T my.file https://my.linx/upload` ?

If you sometimes upload using the api instead of the web page, would you still want the default expiry? Meaning if yes, and you want to upload with no expiry, you'd have to specifically add the `Linx-Expiry: 0` header

**Comment by wxcafe** [comment]
I was only thinking about the upload page as I've never used the API but if implemented yeah it should affect both imo

**Comment by SuperTux88** [comment]
Yes, I'd be happy if I'd be able to set a default for what the upload page has selected in the dropdown too. Don't need to change the API.

**Comment by craSH** [comment]
Personally I think the default in the UI is more important, as I feel any API use would typically be more explicit in specifying parameters. Thanks!
